src/style.css

```diff
@@ -1 +1,56 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src:
+    url('/src/fonts/Avenir-Book.ttf') format('truetype'),
+    url('/src/fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  margin: 20px 8px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  cursor: pointer;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  box-shadow: 0 3px 8px #3d4e611a;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+}
+
+.search-bar__input--big {
+  background-image: url(/src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-x: 26px;
+  background-position-y: center;
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.search-bar__input--small {
```

Comment:
There is no need to create two different modifiers. You can make one of search bars the default one, and add modifier only for another one